index.html

```diff
@@ -78,6 +78,6 @@
 
     <noscript>Please enable JavaScript to view this website.</noscript>
 
-    <script type="module" src="src-js/components/app-index.js"></script>
+    <script type="module" src="src/components/app-index.ts"></script>
```

Comment:
If the `index.html` import the TypeScript file, the build process stops working...